src/irc.py

```diff
@@ -166,17 +166,16 @@ def _generate_app_registration_local(
             "/bin/bash",
             "-c",
             f"[[ -f {IRC_BRIDGE_REGISTRATION_FILE_PATH} ]] || "
-            f"matrix-appservice-irc -r -f {IRC_BRIDGE_REGISTRATION_FILE_PATH}"
-            f" -u https://{matrix.host}:{IRC_BRIDGE_HEALTH_PORT} "
+            f"snap run matrix-appservice-irc -r -f {IRC_BRIDGE_REGISTRATION_FILE_PATH}"
+            f" -u https://{matrix.homeserver}:{IRC_BRIDGE_HEALTH_PORT} "
```

Comment:
This is setting how the IRC bridge will connect to Matrix, right? What is the port?

Reply:
That's well spotted and that's for the next PR - this here tells Matrix where to talk to the bridge, so the port is right (it is the bridge one), but the host isn't ;(
I need to add an ingress and replace that snippet.......

Reply:
Fixed by using only matrix.homeserver